packages/kubernetes-1.15/kubelet.service

```diff
@@ -16,21 +16,8 @@ ExecStartPre=/usr/bin/host-ctr \
     --namespace=k8s.io \
     pull-image \
     --source=${POD_INFRA_CONTAINER_IMAGE}
-ExecStart=/usr/bin/kubelet \
-    --cloud-provider aws \
-    --config /etc/kubernetes/kubelet/config \
-    --kubeconfig /etc/kubernetes/kubelet/kubeconfig \
-    --container-runtime=remote \
-    --container-runtime-endpoint=unix:///run/dockershim.sock \
-    --containerd=/run/dockershim.sock \
-    --network-plugin cni \
-    --root-dir /var/lib/kubelet \
-    --cert-dir /var/lib/kubelet/pki \
-    --volume-plugin-dir /var/lib/kubelet/plugins/volume/exec \
-    --node-ip ${NODE_IP} \
-    --node-labels "${NODE_LABELS}" \
-    --register-with-taints "${NODE_TAINTS}" \
-    --pod-infra-container-image ${POD_INFRA_CONTAINER_IMAGE}
+# Must be overridden by a drop-in file or `kubelet` won't start
+ExecStart=/usr/bin/false
```

Comment:
Maybe I missed it, but where does this get overridden?

Reply:
The first `ExecStart=` line in `kubelet-exec-start.conf` clears out this value, and the second one sets it.